packages/mint-components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/mint-components",
   "title": "Mint Components",
-  "version": "2.0.5",
+  "version": "2.0.6-0",
```

Comment:
You need to manually set the version and deploy before I approve the PR as there are no changesets here